lib/dslash_policy.cuh

```diff
@@ -1862,8 +1603,6 @@ namespace quda
     QUDA_FUSED_ZERO_COPY_PACK_GDR_RECV_DSLASH,
     QUDA_DSLASH_FUSED_PACK,
     QUDA_DSLASH_FUSED_PACK_FUSED_HALO,
-    QUDA_DSLASH_ASYNC,
-    QUDA_FUSED_DSLASH_ASYNC,
     QUDA_DSLASH_NC,
```

Comment:
Are there any hardcoded comparisons of dslash policies with integers that may bork because the integer value of `QUDA_DSLASH_NC` has changed?

Reply:
Nop, and in fact `QUDA_DSLASH_NC` isn't used at all any more and is another legacy policy that could be deleted (It's from the original Mobius implementation that Hyung-Jin wrote).  I figured it could have some use at some point to have a no-comms policy though, so kept it for now.